pyproject.toml

```diff
@@ -107,7 +96,7 @@ profile = "black"
 multi_line_output = 3
 
 [tool.ruff]
-line-length = 115
+line-length = 100
```

Comment:
My editor would just reformat every file b/c this doenst match black's config above, which is what the makefile uses